packages/core/src/components/banner/banner.tsx

```diff
@@ -0,0 +1,35 @@
+import { Component, h, Prop } from '@stencil/core';
+import clsx from 'clsx';
+
+@Component({
+  tag: 'dso-banner',
+  styleUrl: 'banner.scss',
+  shadow: true
+})
+export class Banner {
+  @Prop()
+  status!: 'warning' | 'danger';
+
+  private static statusMap = new Map<string, string>([
+    ['warning', 'Onderhoudsmelding:'],
+    ['danger', 'Storingsmelding:']
+  ]);
+
+  render() {
+    return (
+      <section class={clsx('banner dso-banner', `alert-${this.status}`)} role="alert">
```

Comment:
Waarom staat hier de class `banner` en `dso-banner`? Zie de referentiefiles, markup wordt anders gegenereerd

Reply:
En moeten we niet iets met die `role="alert"` zoals we dat in het Alert issue hebben gedaan? @bramhoosemans ?

Reply:
In het issue is dat een van de punten die je had aangegeven:

```
Ondanks dat dit component geen dso- prefix heeft is het wel een DSO component. Tegelijkertijd met deze change gaan we ook .dso-banner ondersteunen. Tijdens de verhuizing van Nunjucks naar Stencil Component krijg de root div ook de class dso-banner.
```

Reply:
Of bedoel je juist dat `.banner` en `.dso-banner` nooit bij elkaar kunnen/mogen staan. Aangezien `.banner` deprectated gaat zijn

Reply:
Ja dat, en er staat ook dat we `.banner` deprecaten door 'm rood te maken